Makefile

```diff
@@ -147,21 +147,6 @@ deps:
 
 # artifacts
 
-# Regenerate algod swagger spec files
-ALGOD_API_SWAGGER_SPEC := daemon/algod/api/swagger.json
-ALGOD_API_FILES := $(shell find daemon/algod/api/server/common daemon/algod/api/server/v1 daemon/algod/api/spec/v1 -type f) \
-	daemon/algod/api/server/router.go
-ALGOD_API_SWAGGER_INJECT := daemon/algod/api/server/lib/bundledSpecInject.go
-
-# Note that swagger.json requires the go-swagger dep.
-$(ALGOD_API_SWAGGER_SPEC): $(ALGOD_API_FILES) crypto/libs/$(OS_TYPE)/$(ARCH)/lib/libsodium.a
-	cd daemon/algod/api && \
-		PATH=$(GOPATH1)/bin:$$PATH \
-		go generate ./...
-
-$(ALGOD_API_SWAGGER_INJECT): deps $(ALGOD_API_SWAGGER_SPEC) $(ALGOD_API_SWAGGER_SPEC).validated
```

Comment:
@algochoi There are a couple more references to `ALGOD_API_SWAGGER_INJECT` not shown in the diff.  Are they left intentionally?

I assume the remaining references should be deleted.  In case you agree, here's a git patch:  
[diff.txt](https://github.com/algorand/go-algorand/files/10172506/diff.txt).

Additionally + optionally:
* I'd consider renaming `rebuild_swagger` to `kmd-rebuild-swagger` or similar to help clarify its intent.  The command _only_ impacts kmd now.
* _If_ you rename, note there's several references - see `git grep rebuild_swagger`.

Reply:
@algochoi Resolving - Thanks for bringing in the feedback.